vllm/engine/llm_engine.py

```diff
@@ -338,6 +338,11 @@ def __init__(
                 "vllm.llm_engine",
                 self.observability_config.otlp_traces_endpoint)
 
+        tokenizer_group = self.get_tokenizer_group()
+
+        def get_tokenizer_for_seq(sequence: Sequence) -> AnyTokenizer:
+            return tokenizer_group.get_lora_tokenizer(sequence.lora_request)
```

Comment:
add a comment saying that don't capture `self` in this function?

Reply:
Done